include/utils.h

```diff
@@ -146,7 +148,11 @@ class Time {
 
   static Time Now() {
     Time t;
-    gettimeofday(&t.tv_, NULL);
+    auto now = std::chrono::high_resolution_clock::now();
```

Comment:
Why do we need unused `now` here?

Reply:
oops yeah I intended to replace the call belowe with `high_resolution_clock`. will test and fix it!

Reply:
I keep forgetting that `high_resolution_clock` is in terms of ticks and hard to convert to an actual wall clock time. The resolution of `system_clock` is nanoseconds everywhere I've tested though, so I think that should be fine. I deleted the unused line.